integrated_channels/moodle/client.py

```diff
@@ -150,6 +151,100 @@ def _get_access_token(self):
                 response.status_code
             )
 
+    @moodle_request_wrapper
+    def _get_enrolled_users(self, course_id):
+        """
+        placeholder
+        """
+        params = {
+            'wstoken': self.token,
+            'wsfunction': 'core_enrol_get_enrolled_users',
+            'courseid': course_id,
+            'moodlewsrestformat': 'json'
+        }
+        user_response = requests.get(
```

Comment:
yeah, I'm actually confused as to why this is succeeding... it should be a post. Agreed on the usage of the generalized wrapper.

Reply:
Tested locally- Seems like making both a post and get request to Moodle succeed and return the expected list of user enrollments? Coooooool. In that case, since we already have the `_post` wrapper method, perhaps I should just use that?

Reply:
yeah, this entire method can be simplified down to 
```
    @moodle_request_wrapper
    def _get_enrolled_users(self, course_id):
        """
        placeholder
        """
        params = {
            'wstoken': self.token,
            'wsfunction': 'core_enrol_get_enrolled_users',
            'courseid': course_id,
            'moodlewsrestformat': 'json'
        }
        return self._post(params)
```

Reply:
(finally taking the placeholder out now)